CrewChiefV4/App.config

```diff
@@ -100,14 +100,11 @@
       <setting name="custom_device_guid" serializeAs="String">
         <value />
       </setting>
-      <setting name="r3e_launch_params" serializeAs="String">
```

Comment:
As a rule, commandline options for Steam games should be specified within Steam by right-clicking on the title, going to properties, and setting the launch options there. Running via the Steam AppID should be the preferred method for Steam titles anyway.

Consolidating the settings helps reduce the clutter in the Properties page, especially with so many new users coming onboard for rFactor 1 support, I'm just trying to stem the tide of forum posts for support on simple stuff.

I agree that this PR should stick to just rF1 fixes though, and I'll change the setting names back to be consistent with R3E. Doing a piece-meal consolidation of the Properties page is likely to cause more harm than good at this stage anyway, and it's simple enough to do later.